File: switch.js
let nomeHeroi = "Homem de ferro";
let XP = 8002;

    switch(true) {
        case (XP < 1000):
            XP = "Ferro";    
            console.log("O Herói de nome " + nomeHeroi + " está no nível de " + XP);
            break;
        case (XP >= 2001 && XP <= 5000):
            XP = "Bronze";
            console.log("O Herói de nome " + nomeHeroi + " está no nível de " + XP);
            break;
        case (XP >= 6001 && XP <= 7000):
            XP = "Ouro";    
            console.log("O Herói de nome " + nomeHeroi + " está no nível de " + XP);
            break;
        case (XP >= 7001 && XP <= 8000):
            XP = "Platina";
            console.log("O Herói de nome " + nomeHeroi + " está no nível de " + XP);
            break;
        case (XP >= 8001 && XP <= 9000):
            XP = "Ascendente";
            console.log("O Herói de nome " + nomeHeroi + " está no nível de " + XP);
            break;
        case (XP >= 9001 && XP <= 10000):
            XP = "Imortal";
            console.log("O Herói de nome " + nomeHeroi + " está no nível de " + XP);
            break;
        case (XP >= 10001):
            XP = "Radiante";
            console.log("O Herói de nome " + nomeHeroi + " está no nível de " + XP);
            break;

        

}

File: switch.java
package br.com.paulo.desafio;

public class Desafio {
    public static void main(String[] args) {
        String nomeHeroi = "Homem de ferro";
        int XP = 8002;
        String nivel = "";

        switch (XP) {
            case 0:
            case 1:
            case 2:
            case 3:
            // ... outros casos até 1000
            case 1000:
                nivel = "Ferro";
                break;
            case 1001:
            case 1002:
            // ... outros casos até 2000
            case 2000:
                nivel = "Bronze";
                break;
            // Adicione mais casos conforme necessário
            default:
                nivel = "Radiante";  // Valor padrão para casos não especificados
        }

        System.out.println("O Heroi de nome " + nomeHeroi + " esta no nivel de " + nivel);
    }
}
